upstairs/src/lib.rs

```diff
@@ -8689,56 +8691,48 @@ pub async fn up_main(
     };
     let tls_context = Arc::new(tokio::sync::Mutex::new(tls_context));
 
-    let mut client_id = 0;
     /*
-     * Create one downstairs task (dst) for each target in the opt
-     * structure that was passed to us.
+     * Create one downstairs task structure (dst) for the three downstairs
+     * tasks.
      */
-    let dst = opt
-        .target
-        .iter()
-        .map(|dst| {
-            /*
-             * Create the channel that we will use to request that the loop
-             * check for work to do in the central structure.
-             */
-            let (ds_work_tx, ds_work_rx) = watch::channel(1);
-            /*
-             * Create the channel used to submit reconcile work to each
-             * downstairs (when work is required).
-             */
-            let (ds_reconcile_work_tx, ds_reconcile_work_rx) =
-                watch::channel(1);
+    let mut dst = Vec::new();
+    for client_id in 0..3 {
+        /*
```

Comment:
This loop is basically the same as before with the changes being:

Always do three client_ids.
The `ds_work_` is now a `mpsc::channel`
We don't store the IP:Port in the `Target` struct any longer, it's now part of the downstairs struct.